COLLABORATOR_GUIDE.md

```diff
@@ -77,13 +77,13 @@ The Website also uses several other Open Source libraries (not limited to) liste
     - [PostCSS Mixins](https://github.com/postcss/postcss-mixins)
     - [PostCSS Import](https://github.com/postcss/postcss-import)
     - [PostCSS Simple Vars](https://github.com/postcss/postcss-simple-vars)
-- [Tailwind][] is used as our CSS Framework and the Foundation of our Design System
+- [Tailwind](https://tailwindcss.com/) is used as our CSS Framework and the Foundation of our Design System
```

Comment:
Why are you changing these links? They're not broken https://github.com/nodejs/nodejs.org/blob/main/COLLABORATOR_GUIDE.md?plain=1#L491-L501

YOu should add next-intl to the footnote links as the others.

Reply:
I thought that I could add them like how Hero Icons and other links added to the collaborator guide